scene/realistic-lighting-and-shadows/src/main/java/com/esri/samples/realistic_lighting_and_shadows/RealisticLightingAndShadowsController.java

```diff
@@ -38,17 +36,15 @@
 import com.esri.arcgisruntime.mapping.view.LightingMode;
 import com.esri.arcgisruntime.mapping.view.SceneView;
 
-import static java.lang.StrictMath.floor;
-
 public class RealisticLightingAndShadowsController {
 
   @FXML private SceneView sceneView;
-  @FXML private Label time;
+  @FXML private Label timeLabel;
   @FXML private Slider timeSlider;
   @FXML private ComboBox<LightingMode> comboBox;
   private Surface surface;
   private Calendar calendar;
-  private SimpleDateFormat dateFormat;
+  private SimpleDateFormat formatted;
```

Comment:
Sorry if I caused any confusion here: I think `dateFormat` is a good variable name for this object. The comments previously were just to suggest to use the phrasing `format the string` instead of `tidy the string`.